src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1001,4 +1001,72 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.ExpressionStatement
         End Function
     End Class
+
+    Partial Class BoundAddRemoveHandlerStatement
+        Implements IEventAssignmentExpression
```

Comment:
In the current state of the world, no. If the Kind spaces of expressions and statements were distinct, it would be easy to make one bound node serve as both an expression statement and as that statement's expression. However, we're not going to do that.

I spent some time trying to come up with a way to make this node an expression statement, and make its expression be something rational, and the best idea I had was to introduce a wrapper node around this node to act as the expression. A better idea would be welcome.